data/products.json

```diff
@@ -377,5 +377,21 @@
         "link": "https://www.remotepass.com/"
       }
     ]
+  },
+  {
+    "name": "Tabnine",
+    "country_code": "IL",
+    "alternatives": [
+      {
+        "name": "Refact",
+        "country_code": "GB",
+        "link": "https://refact.ai/"
+      },
+      {
+        "name": "CodeGeex",
+        "country_code": "CN",
```

Comment:
1. We can put "OSS" in country_code instead of "CN" as the project is fully open source: [CodeGeeX - GitHub](https://github.com/THUDM/CodeGeeX).
2. We can make the X uppercase: CodeGeex > CodeGee**X**